test/integration/plugins/storpool/MigrateVolumeToStorPool.py

```diff
@@ -194,9 +194,12 @@ def setUpCloudStack(cls):
         securitygroup = SecurityGroup.list(cls.apiclient, account = cls.account.name, domainid= cls.account.domainid)[0]
         cls.helper.set_securityGroups(cls.apiclient, account = cls.account.name, domainid= cls.account.domainid, id = securitygroup.id)
 
+        cls.clusters = cls.helper.getClustersWithStorPool(cls.apiclient, cls.zone.id,)
+
         cls.vm = VirtualMachine.create(cls.apiclient,
```

Comment:
@slavkap is there a reason you are not adding these VMs to cls._cleanup? it seems they would leave garbage in an environment if something goes wrong.

Reply:
Hi @DaanHoogland, I've added the account to the `cls._cleanup`. All VMs and volumes are on this account and will be deleted the account is deleted

Reply:
sound good @slavkap . I hope you tried to break that ;) I trust your judgement more than the actual mechanism.

Reply:
@DaanHoogland, sometimes you're missing adding something to be cleaned, or the deletion fails because it's not in the correct order. That's why I found that when you create all on one test account, CS will delete all objects on it correctly. There is always a possibility something to fail :D